en-US/S.xml

```diff
@@ -523,8 +523,12 @@ a programming and technical sense this also means "Run the <command>source</comm
 				<para>
 					Initialism for Secure Shell, a network protocol that allows data exchange using a secure channel. When referring to the protocol, do not use "ssh," "Ssh," or other variants. When referring to the command, use <command>ssh</command>.
 				</para>
-				 <para>
-					Do not use as a verb. For example, instead of "ssh to the remote server," write "Use SSH to connect to the remote server," "Open an SSH connection," or something similar.
+				<para>
+				<emphasis>Do not use as a verb.</emphasis>
```

Comment:
We have several other cases of "do not use a verb" and none of them use `<emphasis>` so maybe go with the flow?